node.gyp

```diff
@@ -152,7 +152,8 @@
 
       'dependencies': [
         'node_js2c#host',
-        'deps/nghttp2/nghttp2.gyp:nghttp2'
+        'deps/nghttp2/nghttp2.gyp:nghttp2',
+        'node-postmortem-metadata'
```

Comment:
Is this the right place to include `node-postmortem-metadata` as a dependency?

Reply:
Seems fine to me but you should add a `#host` at the end.  For consistency, I'd name it `node_postmortem_metadata`, i.e., underscores instead of dashes.

Reply:
I'll make those changes. Should I also rename `gen-postmortem-metadata.py` to `gen_postmortem_metadata.py` (most files on `tools/` are using dashes, but there are some files using underscores)?